patch/series

```diff
@@ -212,6 +212,22 @@ armhf_secondary_boot_online.patch
 0019-dt-bindings-marvell-Add-ARMADA-7K-properties.patch
 0020-dts-marvell-Add-support-for-7020-comexpress.patch
 
+# amd-pensando elba support
+0001-pensando-elba-soc-support.patch
+0002-ELBA-dts-changes.patch
+0003-Interrupt-domain-controllers-for-Elba-ASIC.patch
+0004-UIO-drivers-for-Elba.patch
+0005-Add-Elba-EDAC-support.patch
+0006-i2c-designware-stuck-bus-recovery.patch
+0007-i2c-Add-Lattice-RD1173-I2C-controller-driver.patch
+0008-spi-dw-elba-chipselect.patch
+0009-pensando-cpld-spidev.patch
+0010-Handle-SError-interrupt-of-type-Decode-Error.patch
+0011-mmc-cadence-elba-support.patch
+0012-drivers-hwmon-support-LTC3888.patch
+0013-hwmon-pmbus-Add-support-for-tps53659.patch
+0014-drivers-reset-Add-emmc-hardware-reset.patch
+
```

Comment:
can you check what is this amd-pensando-elba-soc-support.patch file, is it used?

Reply:
This patch was initially present which had all the changes in a single patch. We no longer need this.. I will delete

Reply:
Deleted